config/locales/en.yml

```
@@ -248,6 +250,11 @@ en:
     not_found_agent: Agent with id %{id}
     not_used: Not used
     ontology_not_valid: 'ontology is not valid, here are the errors: '
+    profile:
+      contributed_in: Has a role in
+      metadata: Metadata
+      statistics: Statistics
+      no_contributions: This agent has no contributions in %{portalName} 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)